h/services/annotation_authority_queue.py

```diff
@@ -0,0 +1,105 @@
+import json
+import logging
+from dataclasses import dataclass
+
+from celery import Celery
+
+from h.services.annotation_read import AnnotationReadService
+
+LOG = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True)
+class AuthorityQueueConfiguration:
+    """Celery setting for an authority specific queue."""
+
+    broker_url: str
+    queue_name: str
+    task_name: str
+
+
+class AnnotationAuthorityQueueService:
+    """A service to publish annotation events to authority specific queues."""
+
+    def __init__(
+        self,
+        authority_queue_config_json: str | None,
+        annotation_read_service: AnnotationReadService,
+        annotation_json_service,
+    ):
+        self._authority_queue_config = self._parse_authority_queue_config(
+            authority_queue_config_json
+        )
+        self._annotation_read_service = annotation_read_service
+        self._annotation_json_service = annotation_json_service
+
+    def publish(self, event_action: str, annotation_id: str) -> None:
+        annotation = self._annotation_read_service.get_annotation_by_id(annotation_id)
+        authority_queue_config = self._authority_queue_config.get(annotation.authority)
+        if not authority_queue_config:
+            return
+
+        if event_action != "create" or not annotation.mentions:
```

Comment:
Will `event_action != "create"` cover annotations that are edited? Or are you addressing editions separately?